tests/utilities/test_warnings.py

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
accidentally here?

Reply:
wdym? This is the test for the changes in this PR

Reply:
what is the shell bang for if this is a python script?
if this is crucial to the fix or how tests are executed it would be good to learn why so we can remember in the future.

Reply:
aah i'm blind

bad copy pasta

```suggestion
```